### ISSUE TITLE
Running bun test fails when post template is moved to post.component.html

### ISSUE THREAD

**Issue by NateRadebaugh** [sp90/treaty#13]
I realize this is a bun issue and not unique to this project, but it looks like you're the only project on all of github who are currently trying to run angular component tests using bun.

Theoretically should be possible using [bun's plugins](https://bun.sh/docs/runtime/plugins), but it's not clear to me if that's the right approach or even if that's the issue.

> Did you run and wait for 'resolveComponentResources()'?

Is this something you've looked into resolving for this project?

**Comment by sp90** [comment]
To be honest its been a while I had issues with various things and I had bun running tests - this issue is a lot of versions old so haven't looked into it lately 

More than welcome to fire a PR or post new issues, i'll get around to it soon

**Comment by Jordan-Hall** [comment]
Their issues with  plugins. Bun v1.2 will be better time to properly relook into it

**Comment by sp90** [comment]
@Jordan-Hall yeah also had jarred respond on the angular issue so hope they include fixing the angular build in v1.2
